### PR TITLE
 Add support to recently deleted global var for multi-indexer

### PR DESCRIPTION
**Pull request by fernandog**
IMO show object must have this unique identifier as an attribute, example 'tvmaze1234'

With new api V2, we will need it in mako and in JS, so it would be good to have in the show object
ping @OmgImAlexis 

Fixes https://github.com/pymedusa/Medusa/issues/2073

### CODE REVIEW

File: medusa/show_queue.py [hunk 1]
```diff
@@ -242,7 +243,7 @@ def removeShow(self, show, full=False):
         self.add_item(queue_item_obj)
 
         # Show removal has been queued, let's updaste the app.RECENTLY_DELETED global, to keep track of it
```
Comment (duramato) [inline medusa/show_queue.py]: `updaste` -> `update`

Comment (fernandog) [inline medusa/show_queue.py]: ops

**Comment by p0psicles** [comment]
Yeah that does make sense. Although I don't think tvmaze1234 is meant as a unique identifier. Or at least it is now. But it's more a kind of slug. And at some time, we should start using the internal tv_show row_id or something similar. But until then, I think it makes sense to use the slug for this.

**Comment by p0psicles** [comment]
Let's have a discussion on this. We should do that short time.
If we approve this one, can you pick it up again, if we make the decision to add the "slug" to the series object.

**Comment by fernandog** [comment]
i tried to add the slug but i got confused when/where it should be added. in the __init__ or a static method?

**Comment by p0psicles** [comment]
Let's discuss it first, then see how to implement it.

**Comment by ratoaq2** [comment]
Create a read only property in Series object that returns the indexer name. And another one that returns this indexer slug id

**Comment by fernandog** [comment]
If later we decide to change slug pattern, just need to change the property

**Comment by fernandog** [comment]
Tested this with backlogged episodes count in editShow and it's working ok. also in displayShow

ping @OmgImAlexis

**Comment by fernandog** [comment]
anyone have an alternative idea to @ratoaq2 idea of a property in Series object?

**Comment by fernandog** [comment]
Renamed to indexer_slug as @ratoaq2 requested

**Comment by labrys** [comment]
We should eventually remove all the various types of indexer_id, indexer combinations and even indexer_slug and replace with a simple ID class.  e.g.:
```python
class SeriesID:
    def __init__(self, source, ident):
        self.source = source
        self.id = ident

    def __repr__(self):
        return '{cls}({source}, {id})'.format(cls=self.__class__.__name__, source=self.source, id=self.id)

    def __str__(self):
        return '{source}_id {id}'.format(source=self.source, id=self.id)

    @property
    def slug(self):
        return '{source}{id}'.format(source=self.source, id=self.id)

>>> series = SeriesID('tvdb', 1234)
>>> series
SeriesID(tvdb, 1234)

>>> 'The series is {series}'.format(series=series)
'The series is tvdb_id 1234'

>>> '/series/{series}'.format(series=series.slug)
'/series/tvdb1234'
```

**Pull request merged**